src/omp/backend.f90

```diff
@@ -383,6 +383,8 @@ subroutine vecadd_omp(self, a, x, b, y)
     real(dp), intent(in) :: b
     class(field_t), intent(inout) :: y
 
+    y%data = a*x%data + b*y%data ! fixme
```

Comment:
This is a temporary implementation necessary for the `omp` test

Reply:
maybe we should create an issue about it then